src/style.css

```diff
@@ -1 +1,72 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  margin: auto auto auto 0;
```

Comment:
remove margin for logo, it should be positioned due to flex and padding on header